include/Engine/Rendering/SpriteJob.h

```diff
@@ -48,6 +48,24 @@ struct SpriteJob : RenderJob
 
         FillColor = fillColor;
         FillPercentage = fillPercentage;
+
+        glm::vec3 scale = Transform::AbsoluteScale(world, cSprite.EntityID);
+
+        if((bool)cSprite["KeepRatioX"] == true) {
+            ScaleX = scale.x;
+        }
+        if ((bool)cSprite["KeepRatioY"] == true) {
+            ScaleY = scale.y;
+        }
+        if((bool)cSprite["KeepRatio"] == true) {
+            if(scale.y >= scale.x) {
+                ScaleY = (scale.x)/(scale.y);
+                ScaleX = 1;
+            } else {
+                ScaleY = 1;
+                ScaleX = (scale.x)/(scale.y);
```

Comment:
It seems that if "KeepRatio" is true, it will write to ScaleX, and Y, even if "KeepRatioX" also did. So it doesn't matter if KeepRatioX is true or not if "KeepRatio" is. Would it make sense to have both enabled? If so the "KeepRatio" branch should take that into account.